src/AppBundle/Controller/SurveyPopupController.php

```diff
@@ -11,7 +11,7 @@
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
```

Comment:
Since `Controller` has been replaced by `BaseController`, the import can be removed.